core/src/test/java/org/bitcoinj/core/ECKeyTest.java

```diff
@@ -239,6 +239,19 @@ public void verifyMessage() throws Exception {
         assertEquals(expectedAddress, gotAddress);
     }
 
+    @Test
```

Comment:
Thanks for the test. But shouldn't it test for a specific recovery id? Currently it just asserts that the recovery id is between 0 and 3.

Is there a test vector in a specification, or in a different implementation perhaps?

Reply:
It would certainly be ideal to test for a specific recovery ID. I'll see what I can dig up.